docs/bundles/helm.md

```diff
@@ -0,0 +1,36 @@
+# Helm Bundle Spec
+
+## Overview
+
+This document is meant to define the helm bundle format as a reference for those publishing helm bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [Bundle](https://github.com/operator-framework/rukpak#bundle) resource.
+
+A helm bundle is a [helm chart](https://helm.sh/docs/chart_template_guide/getting_started/#charts).
+The helm chart can be created by [helm create](https://helm.sh/docs/helm/helm_create/).
+
+The current helm bundle format name is the `helm+v3` that
+combines the type of bundle (helm) with the current helm format version (v3).
+
+## Example
+
+For example, below is an example of a file tree in a "helm+v3" bundle.
+
+```tree
+hello-world
```

Comment:
As of https://github.com/operator-framework/rukpak/pull/516, we support helm chart bundles with or without a root chart directory. So at least the helm provisioner supports both of these:
```
<bundleRoot>
└── nginx
    ├── Chart.yaml
    ├── charts
    ├── templates
    │   ├── NOTES.txt
    │   ├── _helpers.tpl
    │   ├── deployment.yaml
    │   ├── hpa.yaml
    │   ├── ingress.yaml
    │   ├── service.yaml
    │   ├── serviceaccount.yaml
    │   └── tests
    │       └── test-connection.yaml
    └── values.yaml
```

and

```
<bundleRoot>
├── Chart.yaml
├── charts
├── templates
│   ├── NOTES.txt
│   ├── _helpers.tpl
│   ├── deployment.yaml
│   ├── hpa.yaml
│   ├── ingress.yaml
│   ├── service.yaml
│   ├── serviceaccount.yaml
│   └── tests
│       └── test-connection.yaml
└── values.yaml
```

IMO, we should specify in the spec that implementations MUST support both of these.

Reply:
The contents of the tgz file generated by the `helm package` command has the first format.  So the current implementation of the helm provisioner is only supporting the first format.  We can create a new issue to support the second tree and update the docs/bundle/helm.md.  Is it OK?

Reply:
I added support for the second in that PR I linked so that `rukpakctl run my-bd ./<chartDir>` would work.

Reply:
I see.  I'll add the second tree.